config/capi/capi.yml

```diff
@@ -55,7 +55,7 @@ uaa:
     cf_api_controllers:
       secret_name: cf-api-controllers-client-secret
     cf_api_backup_metadata_generator:
-      secret_name: cf-api-backup-metadata-generator-secret
+      secret_name: cf-api-backup-metadata-generator-client-secret
```

Comment:
This needs to match this:
https://github.com/cloudfoundry/cf-for-k8s/pull/559/files#diff-8d0ec3e11b92f104c5b1e83ad6bb3d0bfe0d005b976a1c1b31e29af98690093fR127